scenariorun/src/main/kotlin/com/cosmotech/scenariorun/azure/ScenarioRunServiceImpl.kt

```diff
@@ -149,6 +151,17 @@ internal class ScenarioRunServiceImpl(
     }
   }
 
+  private fun deleteScenarioLastRunEvent(scenarioRun: ScenarioRun) {
+    var scenario =
+        scenarioApiService.findScenarioById(
+            scenarioRun.organizationId!!, scenarioRun.workspaceId!!, scenarioRun.scenarioId!!)
+
+    val lastRunId = scenario.lastRun!!.scenarioRunId
+    scenario.lastRun = null
```

Comment:
If you remove `lastRun` information, set the scenario status to `Created` also. 
In order to be consistent, if a scenario has no `lastRun`, it means that it hasn't been ran, so the scenario status should be "`Created`" instead of "`Failed`, `Successful`,..."